### PR TITLE
Make the alb target info available for use in autoscaling policies

### PR DESCRIPTION
**Pull request by byujol**

### CODE REVIEW

**Review by blakemorgan — CHANGES_REQUESTED** [review]
A few things:

1. You need to update the version number. 3.1.2 -> 3.1.3 (you can do this with a ctrl+f search)
2. You need to update the README to list the new output.
3. Like Josh said in Slack, the workflows failing aren't something you did.

**Comment by GaryGSC** [comment]
We fixed (3) in #17.

File: outputs.tf [hunk 1]
```diff
@@ -26,6 +26,10 @@ output "alb" {
   value = aws_alb.alb
 }
 
+output "alb_target_group" {
+  value = aws_alb_target_group
```
Comment (joshgubler) [inline outputs.tf:30]: This needs to include the resource identifier, not just the resource class. It should be `aws_alb_target_group.something`.

Comment (joshgubler) [reply outputs.tf:30]: Look in main.tf to figure out what the `something` is.

**Pull request closed without merging**